packages/terra-application-header/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Application Header
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-application-header.svg)](https://www.npmjs.org/package/terra-application-header)
+[![Build Status](https://travis-ci.org/cerner/terra-framwork.svg?branch=master)](https://travis-ci.org/cerner/terra-work)
```

Comment:
Missing an `e` here: `terra-framwork.svg` => `terra-framework.svg`
Missing the word `frame` here: `travis-ci.org/cerner/terra-work` => `travis-ci.org/cerner/terra-framework`